firebase-firestore/src/test/java/com/google/firebase/firestore/spec/SpecTestCase.java

```diff
@@ -264,8 +264,7 @@ protected void specTearDown() throws Exception {
    */
   private void initClient() {
     localPersistence = getPersistence(garbageCollectionEnabled);
-    // TODO(index-free): Update to index-free query engine when it becomes default.
```

Comment:
There's one other TODO which I think is actually just stale / wrong.

https://github.com/firebase/firebase-android-sdk/blob/209285735903837d12d0af476132f6415edf9e95/firebase-firestore/src/main/java/com/google/firebase/firestore/local/SQLiteSchema.java#L49

Reply:
This was going to go away with my schema backfill PR. But instead that went away...